app/server/src/routes/space/space.controller.js

```diff
@@ -17,8 +19,13 @@ const SpaceController = {
         tags,
         image
       );
-      creator.created_spaces.push(space._id);
-      await creator.save();
+      // {user} created a new space called {space.name}, {date.now-space.createdAt} ago.
+      let activity_body = `${creator.name} ${creator.surname} created a new space called ${space.name}, {timeDiff}.`;
+      let activity_data = {
+        body : activity_body,
+        space: space._id,
+      }
```

Comment:
This activity uses the same field (space) in the data as create enrollment. It's hard to distingush, it seems they will have to match patterns of bodies in the front-end.

Reply:
Maybe I should add activity type field ? They can do string matching with that.

Reply:
Yes, good idea. If we give the activity type, they can even create the body string themselves.

Reply:
We can send the time difference string as a field.

Reply:
Makes sense, a little cumbersome but okay :d

Reply:
Or I can directly embed the links since the work is same , saves me a lot of time

Reply:
You can, since both ends use markdown, you can add links as markdown.

Reply:
The problem is are they going to use markdown text on feed. If not how do we show links. html?